boards/arm/otterpill/otterpill.dts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2023 Jonas Otto
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f0/stm32f072X8.dtsi>
+#include <st/f0/stm32f072c(8-b)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "OtterPill";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_status: led_status {
+			gpios = <&gpiob 13 GPIO_ACTIVE_HIGH>;
+			label = "Status LED";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		dfu_button: button {
+			label = "DFU";
+			gpios = <&gpioc 13 GPIO_ACTIVE_HIGH>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	aliases {
+		led0 = &led_status;
+		sw0 = &dfu_button;
+		watchdog0 = &iwdg;
+	};
+
+	arduino_nano_header: connector {
```

Comment:
Please use a dedicated arduino header dtsi file. See nucleo boards for instance.

Reply:
done